main.go

```diff
@@ -110,10 +105,6 @@ func (p *Proxy) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 
 	for d.NextBlock(0) {
 		switch d.Val() {
-		case "management_api":
```

Comment:
Having a `switch` block without a `case` statement seems a little strange to me.  Are you sure you need it now? Would it be better just to have it return an error inside the for loop?

Reply:
Next PR will be a refactor PR, I'll include this in that one